pkg/kubefed2/federate/disable.go

```diff
@@ -47,11 +47,19 @@ var (
 		--host-cluster-context flag otherwise.`
 
 	disable_example = `
-		# Disable propagation of the Service type
-		kubefed2 federate disable Service
-
-		# Disable propagation of the Service type and delete API resources
-		kubefed2 federate disable Service --delete-from-api`
+		# Disable propagation of the kubernets API type 'Deployment', named
+		in FederatedTypeConfig as 'deployment.apps'
+		kubefed2 disable deployment.apps
+
+		# Disable propagation of the kubernets API type 'Deployment', named
+		in FederatedTypeConfig as 'deployment.apps', and delete corresponding
+		Federated API resources
+		kubefed2 disable deployment.apps --delete-from-api
+
+		Note: The NAME needed in disable command is the name of the FederatedTypeConfig
```

Comment:
I don't think it should be necessary to document this here.  The name of an ftc can be discovered from the API - it doesn't need to be invented.